src/ui/utils/devtools-toolbox.js

```diff
@@ -65,7 +65,6 @@ export class DevToolsToolbox {
     await panel.open();
 
     this.panels[name] = panel;
-    store.dispatch(actions.setSelectedPanel(name));
```

Comment:
We don't need to open up the panel every time we start it. Since we front load opening the console and the debugger, this would cause the toolbox to flicker very quickly before setlling on the actual selected panel. (console -> debugger -> selectedPanel)